### ISSUE TITLE
Please provide RPM

### ISSUE THREAD

**Issue by Yamakasi** [gluster/cockpit-gluster#21]
Hi is it possible for you to provide a an RPM based on latest PR ?

**Comment by skunkr** [comment]
Hi, I am searching for a rpm as well.  Installing the repos as in the tutorial does not allow me to get the package...
 yum install cockpit-gluster
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.netweaver.uk
 * centos-gluster5: centos.mirroring.pulsant.co.uk
 * epel: mirrors.coreix.net
 * extras: mirrors.coreix.net
 * updates: mirror.as29550.net
No package cockpit-gluster available.
Error: Nothing to do

Any ideeas ?  Thank you !

**Comment by dlabreu** [comment]
+1  I Also Would love have this as an RPM. I assume this is possible as in the Ovirt-Node you can create a Gluster Cluster from Cockpit.
https://ovirt.org/documentation/gluster-hyperconverged/chap-Deploying_Hyperconverged.html

**Comment by sjkeerthi** [comment]
Its a very simple I just did it on my CentOS 7.x 

# yum install gcc rpm-build 
# git clone https://github.com/gluster/cockpit-gluster.git
# cd cockpit-gluster
# mkdir dist
# make rpm

After that you can see the rpm in cockpit-gluster/rpm_build/RPMS/noarch
cockpit-gluster-0.0.1-1.noarch.rpm

Try and let me know if any issues

**Comment by Yamakasi** [comment]
> Its a very simple I just did it on my CentOS 7.x
> 
> `# yum install gcc rpm-build `
> `# git clone https://github.com/gluster/cockpit-gluster.git`
> ` # cd cockpit-gluster`
> ` # mkdir dist`
> ` # make rpm`
> 
> After that you can see the rpm in cockpit-gluster/rpm_build/RPMS/noarch
> cockpit-gluster-0.0.1-1.noarch.rpm
> 
> Try and let me know if any issues

True, but it took quite some time, and you miss mostly some packages so if we have one buildsystem and available RPM's I think it's more solid ?

**Comment by mostafabarmshory** [comment]
+1
